pyaerocom/scripts/cli.py

```diff
@@ -41,11 +41,11 @@ def clearcache():
 
     delete = typer.confirm("Are you sure you want to delete all cached data objects?")
     if delete:
-        print("OK then.... here we go!")
+        print("Okay then.... Here we go!")
```

Comment:
do we need this print?

Reply:
Not sure if we need it, but some confirmation of the user choice to delete the cache should be given.